test/timearray.jl

```diff
@@ -480,7 +480,7 @@ end
 
 @testset "show methods don't throw errors" begin
     io = IOBuffer()
-    let str = sprint(show, cl)
+    let str = sprint(summary, cl)
         out = "500×1 TimeArray{Float64,1,Date,Array{Float64,1}} 2000-01-03 to 2001-12-31"
```

Comment:
It seems that tests are failing in 1.6 because `Array{Float64,1}` is now shown as `Vector{Float64}` and `Array{Float64,2}` as `Matrix{Float64}`.

Reply:
Maybe in `test/timearray.jl` we can add a string to interpolate in the output depending on the value of `VERSION`. Something like: 

```julia
dispvec = VERSION < v"1.6" ? "Array{Float64,1}" : "Vector{Float64}"
```

Or maybe even typed: 
```julia
dispvec(T) = VERSION < v"1.6" ? "Array{$(repr(T)),1}" : "Vector{$(repr(T))}"
```

Reply:
ah, fine. :+1:

Reply:
Turns out that this `TimeArray{Float64,1,Date,Array{Float64,1}}` becomes `TimeArray{Float64, 1, Date, Vector{Float64}}` in 1.6.